common/errors.js

```diff
@@ -293,7 +293,7 @@
         var exceptionFlag = false;
 
         // TODO: implement hierarchies of exceptions in ermrestJS and use that hierarchy to conditionally check for certain exceptions
-        function handleException(exception, isDismissible) {
+        function handleException(exception, isDismissible, skipLoginDialog) {
```

Comment:
I couldn't think of a better way to handle this except by adding a new flag to the handleException function.